### DISCUSSION TITLE
请问pytorch版本有要求吗？

### DISCUSSION THREAD

**Discussion by Jianshu-She** [fishaudio/Bert-VITS2#296] [Q&A] [answered]
AttributeError: module 'torch.backends.cuda' has no attribute 'enable_mem_efficient_sdp'，是不是torch版本太低

**Comment by Stardust-minus** [comment, marked as answer]
> AttributeError: module 'torch.backends.cuda' has no attribute 'enable_mem_efficient_sdp'，是不是torch版本太低

代码里有注释，要求>2.0
